templates/questionTemp.js

```diff
@@ -0,0 +1,154 @@
+
+
+
+const baseQuestion = {
```

Comment:
@swetha-aggidevara we should move this inside module of questions and do add comments on each of the object types.